CONTRIBUTING.md

```diff
@@ -37,6 +37,14 @@ can up-vote that issue. This helps us to see what the community wants the most.
 If there does not already exist a feature request for your idea,
 you can [request a feature](https://github.com/shopware/meteor/issues/new).
 
+## Local development
+
+If you develop a feature locally and want to test it within Shopware follow these step:
+
+1. Make changes for the desired package/s
```

Comment:
IMO it makes sense to add a little sentence about yalc and provide a link to it's documentation, for those who aren't familiar with yalc.